Mathlib/FieldTheory/AbsoluteGaloisGroup.lean

```diff
@@ -34,16 +34,25 @@ variable (K : Type*) [Field K]
 
 /-! ### The absolute Galois group -/
 
-/-- The absolute Galois group of `K`, defined as the Galois group of the field extension `K^al/K`,
-  where `K^al` is an algebraic closure of `K`. -/
-def absoluteGaloisGroup := AlgebraicClosure K ≃ₐ[K] AlgebraicClosure K
+/-- The absolute Galois group of `K`, defined as the Galois group of the field extension `K^sep/K`,
+  where `K^sep` is the separable closure of `K`. -/
+def absoluteGaloisGroup := SeparableClosure K ≃ₐ[K] SeparableClosure K
```

Comment:
Can you please show that [AlgEquiv.restrictNormal](https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/Normal.html#AlgEquiv.restrictNormal) gives a canonical isomorphism between the old and new definitions? (And what's the reason to prefer the new one?)
(There's the instance [separableClosure.isGalois](https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/SeparableClosure.html#separableClosure.isGalois), and [IsPurelyInseparable.bijective_restrictDomain](https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/PurelyInseparable.html#IsPurelyInseparable.bijective_restrictDomain) should be helpful.)